src/ruby/qps/proxy-worker.rb

```diff
@@ -155,6 +157,10 @@ def proxymain
     opts.on("-c", "--[no-]use_protobuf_c_extension", "Use protobuf C-extention") do |c|
       options[:c_ext] = c
     end
+    opts.on("-b" "--php_client_bin [FILE]",
```

Comment:
should there be a comma after "-b"?

Reply:
Done. For my own information, is this valid Ruby...? I was still able to run tests despite this problem.